server/src/domain/repositories/search.repository.ts

```diff
@@ -157,7 +157,9 @@ type BaseAssetSearchOptions = SearchDateOptions &
 
 export type AssetSearchOptions = BaseAssetSearchOptions & SearchRelationOptions;
 
-export type AssetSearchOneToOneRelationOptions = BaseAssetSearchOptions & SearchOneToOneRelationOptions;
+export type AssetSearchOneToOneRelationOptions = BaseAssetSearchOptions &
+  SearchOneToOneRelationOptions &
+  SearchRelationOptions;
```

Comment:
Don't add this to this type. Change `SearchOneToOneRelationOptions` to include the stack relation

Reply:
Where did these butt-ugly types even come from. I've been away too long...

Reply:
I've been writing too much Java code lol